Model/IO/Export/VTK_exporter/VTK_Exporter.cpp

```diff
@@ -0,0 +1,70 @@
+#include "VTK_Exporter.h"
+
+void VTK_Exporter::exportHeader()
+{
+	VTK_file << "# vtk DataFile Version 2.0\n";
+	VTK_file << "MeshIS\n";
+	VTK_file << "ASCII\n";
+	VTK_file << "DATASET UNSTRUCTURED_GRID\n";
+}
+
+void VTK_Exporter::exportPoints(const CMR & mesh_data, int numOfPoints)
+{
+	VTK_file << "POINTS " << numOfPoints << " double\n";
+
+	for (int i = 0; i < mesh_data.vertices.size(); i++) {
+		VTK_file << mesh_data.vertices[i][0] << " "
+			<< mesh_data.vertices[i][1] << " "
+			<< mesh_data.vertices[i][2] << "\n";
+	}
+
+	VTK_file << "\n";
+}
+
+void VTK_Exporter::exportCells(const CMR & mesh_data, int numOfElements, int numOfEntries)
```

Comment:
Did you consider using data from mesh_data to obtain values of numOfElements and numOfEntries?